package.json

```diff
@@ -35,11 +36,14 @@
   },
   "devDependencies": {
     "jest": "^27.5.0",
-    "node-gyp": "^8.4.1",
-    "prebuild": "^11.0.3",
+    "node-gyp": "^9.4.0",
+    "prebuild": "^11.0.4",
     "prettier": "^2.5.1",
     "split2": "^4.1.0"
   },
+  "resolutions": {
+    "node-gyp": "^9.4.0"
+  },
```

Comment:
this is required due to rebuild rotting and not upping their dependencies. it currently depends on node-gpy `v6.0.0` which has build issues with python-3.11 (that multiple of the github runners now use)

`v9.0.0` is api compatible so we force it. upstream bug long reported here https://github.com/prebuild/prebuild/issues/286